ex3/ex3-index.js

```diff
@@ -0,0 +1,43 @@
+// 3. Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+// Funcția letterCount() returnează numărul apariții al unei litere trimise ca parametru dintr-un șir dat.
+// Exemplu 1:
+// Input: sir = “Îmi place programarea” litera = “a”
+// Output: 4
+// Exemplu 2:
+// Input: sir = “Vreau să lucrez în IT”, litera = “r”
+// Output: 2
+// Notă: Nu se ține cont de scrierea cu majuscule sau minuscule.
+// Litera A și litera a trebuie numărate la calcularea numărului de apariții pentru A sau pentru a
+//.........................................................................................................
+
+// const countLetterOccurrences = (string, letter) => (typeof string === "string") && (typeof letter === "string") ? for char in string ? count++ : "Invalid input: all input must be string";
+console.clear()
+
+const countLetterOccurrences = (string, letter) => {
+  if (typeof string === "string" && typeof letter === "string") {
```

Comment:
ar fi mult mai frumos sa facem validarea la inceput:

if (typeof string !== "string" || typeof letter === "string") {
  throw "Both arguments must be strings.";
}

si apoi incepem codul fara `if`